sycl/test-e2e/Basic/fill_accessor_pi.cpp

```diff
@@ -0,0 +1,147 @@
+// RUN: %{build} -o %t.out
+// RUN: env SYCL_PI_TRACE=2 %{run} %t.out | FileCheck %s
+
+// the fill_accessor.cpp test checks that the .fill() function is working
```

Comment:
Is there supposed to be another test named fill_accessor.cpp?

Reply:
Oh, I see, it is an existing test. Probably better to avoid mentioning file names as they may be renamed/removed etc.

Reply:
I see the two tests as sisters to each other. I think updating the comment might help avoid confusion.